lib/ssl/src/tls_gen_connection.erl

```diff
@@ -363,12 +364,20 @@ handle_info({ssl_tls, Port, Type, {Major, Minor}, Data}, StateName,
                    ssl_options = #{ktls := true}} = State0) ->
     Len = byte_size(Data),
     handle_info({Protocol, Port, <<Type, Major, Minor, Len:16, Data/binary>>}, StateName, State0);
+handle_info({ErrorTag, Socket, abort, {_Handle, closed}}, StateName,
```

Comment:
For a none "socket" socket I guess the last tuple looks different (might not be a tuple) and ends up in ssl_gen_statem ?

Reply:
Yeah probably, but the ErrorTag would hopefully not match sockets error tags :-)